packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -294,7 +294,13 @@ async function handler(
     })
   );
 
-  const eventTimeZone = eventType.schedule?.timeZone ?? null;
+  const user = eventType.users.filter((user) => user.id === eventType.userId)[0];
+
+  const userSchedule = user.schedules.filter(
+    (schedule) => !user?.defaultScheduleId || schedule.id === user?.defaultScheduleId
+  )[0];
```

Comment:
Using filter here is unnecessary; please use the `find` method instead.

Reply:
Yes! Thanks @anikdhabal !